packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx

```diff
@@ -145,6 +152,37 @@ describe('VitalsBiometricsForm: ', () => {
     expect(muac).toHaveValue(23);
 
     await waitFor(() => user.click(saveButton));
+
+    expect(mockSavePatientVitals).toHaveBeenCalledTimes(1);
+    expect(mockSavePatientVitals).toHaveBeenCalledWith(
+      mockVitalsConfig.vitals.encounterTypeUuid,
+      mockVitalsConfig.vitals.formUuid,
+      mockVitalsConfig.concepts,
+      mockPatient.id,
+      expect.objectContaining({
+        height: '180',
+        midUpperArmCircumference: '23',
+        oxygenSaturation: '100',
+        pulse: '80',
+        respiratoryRate: '16',
+        systolicBloodPressure: '120',
+        temperature: '37',
+        weight: '62',
```

Comment:
Please don't use magic numbers here, move them into variables. Use the same value when typing too (lines 137 - 144).

Reply:
yeah sure